src/domain/app/footer/__tests__/__snapshots__/Footer.test.tsx.snap

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Footer matches snapshot 1`] = `"<footer class=\\"footerWrapper\\"><div class=\\"Koros-module_koros__1r3rg koros\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" aria-hidden=\\"true\\" width=\\"100%\\" height=\\"85\\"><defs><pattern id=\\"koros_basic-1\\" x=\\"0\\" y=\\"0\\" width=\\"106\\" height=\\"85\\" patternUnits=\\"userSpaceOnUse\\"><path transform=\\"scale(5.3)\\" d=\\"M0,800h20V0c-4.9,0-5,2.6-9.9,2.6S5,0,0,0V800z\\"></path></pattern></defs><rect fill=\\"url(#koros_basic-1)\\" width=\\"100%\\" height=\\"85\\"></rect></svg></div><div class=\\"container default\\"><div class=\\"footer\\"><div class=\\"helsinkiLogo\\"></div><div class=\\"copyright\\"><div>© Copyright 2020 • Kaikki oikeudet pidätetään</div><div><a href=\\"https://www.hel.fi/static/liitteet/kanslia/rekisteriselosteet/Kuva/Kuva-EU-Palvelutarjotin.pdf\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">Rekisteriseloste</a></div></div></div></div></footer>"`;
+exports[`Footer matches snapshot 1`] = `"<footer class=\\"footerWrapper\\"><div class=\\"Koros-module_koros__1r3rg koros\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" aria-hidden=\\"true\\" width=\\"100%\\" height=\\"85\\"><defs><pattern id=\\"koros_basic-1\\" x=\\"0\\" y=\\"0\\" width=\\"106\\" height=\\"85\\" patternUnits=\\"userSpaceOnUse\\"><path transform=\\"scale(5.3)\\" d=\\"M0,800h20V0c-4.9,0-5,2.6-9.9,2.6S5,0,0,0V800z\\"></path></pattern></defs><rect fill=\\"url(#koros_basic-1)\\" width=\\"100%\\" height=\\"85\\"></rect></svg></div><div class=\\"container default\\"><div class=\\"footer\\"><div class=\\"helsinkiLogo\\"></div><div class=\\"copyright\\"><div>© Copyright 2020 • Kaikki oikeudet pidätetään</div><div><a href=\\"https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Kuva/Henkil%C3%B6rekisteri%20Palvelutarjotin.pdf\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">Rekisteriseloste</a></div></div></div></div></footer>"`;
```

Comment:
Seems liek Footer snapshot has not been well formatted and now it is really hard to see changes.. Can you change that to this PR?

Reply:
fixed now all enzyme snapshots to testinglibrary snapshots